### PR TITLE
TT-14 create student data form

### PR DESCRIPTION
**Pull request by kBlazewicz**
image

image

### CODE REVIEW

**Review by Paulina100 — APPROVED** [review]
Działa

File: client/src/components/student-data-form/StudentDataForm.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+import { TextField, Button, Card, CardContent } from "@material-ui/core";
+import { StudentData } from "../../types/types";
+import { sendStudentData } from "../../api/api-utils";
+import "./StudentDataForm.css";
+import { Typography } from "@mui/material";
+
+const StudentDataForm: React.FC = () => {
```
Comment (fkitka) [inline client/src/components/student-data-form/StudentDataForm.tsx:8]: React.FC niepotrzebne
https://juhanajauhiainen.com/posts/should-you-use-reactfc-to-type-your-components
TLDR: No. You should probably not.

File: client/src/components/student-data-form/StudentDataForm.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                    size="medium"
+                >
+                    Submit
+                </Button>
+            </CardContent>
+        </Card>
+    );
+};
+
+export default StudentDataForm;
```
Comment (fkitka) [inline client/src/components/student-data-form/StudentDataForm.tsx:115]: https://stackoverflow.com/questions/33305954/typescript-export-vs-default-export
